src/main/kotlin/org/opensearch/replication/action/replay/TransportReplayChangesAction.kt

```diff
@@ -72,7 +73,7 @@ class TransportReplayChangesAction @Inject constructor(settings: Settings, trans
     TransportWriteAction<ReplayChangesRequest, ReplayChangesRequest, ReplayChangesResponse>(
         settings, ReplayChangesAction.NAME, transportService, clusterService, indicesService, threadPool, shardStateAction,
         actionFilters, Writeable.Reader { inp -> ReplayChangesRequest(inp) }, Writeable.Reader { inp -> ReplayChangesRequest(inp) },
-            EXECUTOR_NAME_FUNCTION, false, indexingPressureService, systemIndices) {
+            EXECUTOR_NAME_FUNCTION, false, indexingPressureService, systemIndices, NoopTracer.INSTANCE) {
```

Comment:
Do we have any tracking task for tracer integration?

Reply:
No